cmd/odf/restore/crds.go

```diff
@@ -20,8 +71,54 @@ var deletedCmd = &cobra.Command{
 	},
 	Run: func(cmd *cobra.Command, args []string) {
 		k8sutil.SetDeploymentScale(cmd.Context(), root.ClientSets.Kube, root.OperatorNamespace, "ocs-operator", 0)
-		pkgrestore.RestoreCrd(cmd.Context(), root.ClientSets, root.OperatorNamespace, root.StorageClusterNamespace, args)
-		k8sutil.SetDeploymentScale(cmd.Context(), root.ClientSets.Kube, root.OperatorNamespace, "ocs-operator", 1)
+		// Parse the fully qualified CRD (e.g. "cephclusters.ceph.rook.io").
+		resourceName, groupName, version, err := parseFullyQualifiedCRD(args[0])
+		if err != nil {
+			logging.Fatal(fmt.Errorf("Error parsing CRD type: %v\n", err))
+		}
+		// Construct a new args slice with the resource name as the first argument.
+		newArgs := make([]string, len(args))
+		newArgs[0] = resourceName
+		if len(args) > 1 {
+			newArgs[1] = args[1]
+		}
+		var customResources []pkgrestore.CustomResource
+		if contains(newArgs, "storageclusters") {
```

Comment:
```suggestion
		if contains(newArgs, "storageclusters") {
```
I think we can use `string.Slices` here without https://go.dev/play/p/sj50jslDBUY to make code clean

Reply:
done